### PR TITLE
Various Fixes

### PR DESCRIPTION
**Pull request by Gragt**
The rationale for some of these commits can be found in the commit message when it isn’t implicitly obvious.

### CODE REVIEW

File: src/epub/text/chapter-11.xhtml [hunk 1]
```diff
@@ -85,7 +85,7 @@
...
 			<p>The subject seemed strangely chosen for an infant singer; but I suppose the point of the exhibition lay in hearing the notes of love and jealousy warbled with the lisp of childhood; and in very bad taste that point was: at least I thought so.</p>
 			<p>Adèle sang the canzonette tunefully enough, and with the naivete of her age. This achieved, she jumped from my knee and said, “Now, Mademoiselle, I will repeat you some poetry.”</p>
-			<p>Assuming an attitude, she began “La Ligue des Rats: fable de La Fontaine.” She then declaimed the little piece with an attention to punctuation and emphasis, a flexibility of voice and an appropriateness of gesture, very unusual indeed at her age, and which proved she had been carefully trained.</p>
+			<p>Assuming an attitude, she began “<span epub:type="se:name.publication.poem" xml:lang="fr">La Ligue des Rats</span>: <i xml:lang="fr">fable de La Fontaine</i>.” She then declaimed the little piece with an attention to punctuation and emphasis, a flexibility of voice and an appropriateness of gesture, very unusual indeed at her age, and which proved she had been carefully trained.</p>
```
Comment (acabal) [inline src/epub/text/chapter-11.xhtml]: If `fable de La Fontaine` is a subtitle, then it should not be italicized. Titles in non-English don't get italics.

**Review by acabal — CHANGES_REQUESTED** [review]

**Comment by acabal** [comment]
Looks good, just a minor nit in italics for the title. If that's not a subtitle then italics are fine, but if it is then there should not be italics.

File: src/epub/text/chapter-11.xhtml (hunk 1 above)
Comment (Gragt) [reply src/epub/text/chapter-11.xhtml]: It’s not part of the official poem title in any way. It seems to be there to tell the reader where the poem is from.

Comment (acabal) [reply src/epub/text/chapter-11.xhtml]: I think it is a type of subtitle, see https://www.google.com/books/edition/Fables_de_La_Fontaine/mbABAAAAYAAJ?hl=en&gbpv=1&dq=%22La+Ligue+des+rats%22&pg=PA283&printsec=frontcover

Comment (acabal) [reply src/epub/text/chapter-11.xhtml]: Oops, this link rather: https://www.google.com/books/edition/Fables_de_La_Fontaine/mbABAAAAYAAJ?hl=en&gbpv=1&pg=PR3&printsec=frontcover

Comment (Gragt) [reply src/epub/text/chapter-11.xhtml]: Fair enough. It isn’t part of the official title, but if “a type of” subtitle works for you, it certainly works for me.

**Comment by Gragt** [comment]
I've (force) pushed the change. Let me know if that's okay.

**Comment by acabal** [comment]
Great, thanks!

**Pull request merged**